src/Symfony/Component/JsonPath/JsonCrawler.php

```diff
@@ -474,43 +531,52 @@ private function compare(mixed $left, mixed $right, string $operator): bool
         };
     }
 
-    private function parseCommaSeparatedValues(string $expr): array
+    /*
+     * Transform JSONPath regex patterns to comply with RFC 9535. The main issue is
+     * that '.' should not match \r or \n but should match Unicode line
+     * separators U+2028 and U+2029.
```

Comment:
in case this wasn't in intentional?
```suggestion
    /**
     * Transforms JSONPath regex patterns to comply with RFC 9535.
     *
     * The main issue is that '.' should not match \r or \n but should
     * match Unicode line separators U+2028 and U+2029.
```

Reply:
Wasn't sure which CS to adopt here, but suits me fine! I updated this part with your suggestion